mender/templates/gui-deploy.yaml

```diff
@@ -62,7 +62,7 @@ spec:
 
         env:
         - name: INTEGRATION_VERSION
-          value: "2.7.0"
+          value: "2.7.1"
         - name: MENDER_VERSION
           value: "2.6.0"
```

Comment:
`MENDER_VERSION` and `MENDER_DEB_VERSION` should be set to `2.6.1`.

Reply:
done!